alonzo/impl/src/Cardano/Ledger/Alonzo/FakePlutus.hs

```diff
@@ -109,5 +108,7 @@ data ScriptPurpose
 data Context = Context TxInfo ScriptPurpose
 
 instance IsData Context where
-  toData (Context _ _) = undefined
+  -- toData will be implemented in the Plutus library,
+  -- this is just a FakePlutus hack.
+  toData (Context _ _) = I 0
```

Comment:
This is handled correctly in https://github.com/input-output-hk/cardano-ledger-specs/pull/2242